packages/nimble-components/src/anchor-tab/index.ts

```diff
@@ -28,6 +28,15 @@ export class AnchorTab extends AnchorBase {
      */
     @attr({ mode: 'boolean' })
     public disabled = false;
+
+    /**
+     * Provide our own attribute reflection for the aria-selected attribute.
```

Comment:
We should follow the pattern that fast is doing in aria-global for the comment:

https://github.com/microsoft/fast/blob/archives/fast-element-1/packages/web-components/fast-foundation/src/patterns/aria-global.ts#L13-L23

Reply:
Done